regression-test/suites/nereids_function_p0/scalar_function/A-F.groovy

```diff
@@ -115,8 +115,6 @@ suite("nereids_scalar_fn_1") {
     sql "select current_date() from fn_test"
     sql "select current_time() from fn_test"
     sql "select current_timestamp() from fn_test"
-    // core
-    // qt_sql "select current_timestamp(kint) from fn_test order by kint"
```

Comment:
we have a signature accept one parameter with int type, so this test case need be added back
https://doris.apache.org/docs/dev/sql-manual/sql-functions/date-time-functions/current_timestamp?_highlight=current_